search/includes/classes/class-search.php

```diff
@@ -550,14 +550,14 @@ public function filter__ep_do_intercept_request( $request, $query, $args, $failu
 
 		$timeout = $this->get_http_timeout_for_query( $query, $args );
 
-		$request = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, $timeout, 20, $args );
+		$response = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, $timeout, 20, $args );
```

Comment:
💯 on renaming the variables!

Reply:
It took me more time to understand the code due to this `$request` variable name, so I thought it would be a good idea to rename it for the next one who gets here 😊